### ISSUE TITLE
Mod broken after latest TR update

### ISSUE THREAD

**Issue by mtb-xt** [TED-inc/FabricQuarry#12]
Hi! 

Someone probably already mentioned, but TechReborn was updated today, and now Quarry fails to start with this error:
> Could not find required mod: quarrymod requires {techreborn @ [>=5.0.4-beta]}

The version string for TR changed to 
```
  "id": "techreborn",
  "version": "project.5.0.7-beta+build.107",
```

**Comment by TED-inc** [comment]
I will check it, thank for notify

**Comment by TED-inc** [comment]
Thank for your attention. Fixed in 0.4.2
